addon/components/cookie-banner/component.js

```diff
@@ -0,0 +1,29 @@
+import Ember from 'ember';
+import layout from './template';
+
+/**
+ * @module ember-osf
+ * @submodule components
+ */
+
+ /**
+ * Adds a cookie banner to the page
+ *
+ * Sample usage:
+ * ```handlebars
+ * {{cookie-banner
+ *   addCookie=(action 'addCookie')
+ * }}
+ * ```
+ *
+ * @class dropzone-widget
+ */
+export default Ember.Component.extend({
+    layout,
+
+    actions: {
+        addCookie() {
+            this.addCookie();
```

Comment:
Not sure where to put so here goes:
I sent more screenshots via flow but the short version is: Click accept on banner. Hard reload page: See console error:

<img width="1440" alt="screen shot 2018-07-10 at 4 22 13 pm" src="https://user-images.githubusercontent.com/1322421/42535657-f02fac26-845d-11e8-901b-3b89e825d859.png">

Reply:
After looking into the error a bit it looks like it's something that's already happening on staging2, but it's being sent to Sentry.